src/parser/deserializer.ts

```diff
@@ -45,6 +45,19 @@ export interface DeserializeOptions {
   index?: number;
 
   raw?: boolean;
+  /** allows for opt-out utf-8 validation for all keys or
+   * specified keys. Must be all true or all false.
+   *
+   * @example
+   * ```js
+   * // disables validation on all keys
+   *  validation: { utf8: false }
+   *
+   * // enables validation on specified keys a, b, and c
+   *  validation: { utf8: { a: true, b: true, c: true } }
+   * ```
+   */
+  validation?: { utf8: boolean | Record<string, boolean> };
```

Comment:
We could codify the all true or all false requirement in typescript:
```suggestion
  validation?: { utf8: boolean | Record<string, true> |  Record<string, false> };
```